.github/workflows/ci.yml

```diff
@@ -20,6 +20,15 @@ jobs:
      run-integration-tests: false
      run-lint: true
 
+  proto_lint:
```

Comment:
I thought this is already a part of https://github.com/babylonlabs-io/babylon/pull/358/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR21. Could we double check?

Reply:
I checked, [run-lint](https://github.com/babylonlabs-io/.github/blob/main/.github/workflows/reusable_go_lint_test.yml#L18) is the go lint, but this one is the protobuf lint.